src/components/auth/register/Register.svelte

```diff
@@ -19,133 +25,204 @@
-            {#if usernameValid && usernameAvailable}
-              <span class="label-text-alt text-green-700 dark:text-green-500">
-                This username is available.
-              </span>
-            {:else if !usernameValid}
-              <span class="label-text-alt text-error">
-                This username is invalid.
-              </span>
-            {:else if !usernameAvailable}
-              <span class="label-text-alt text-error">
-                This username is unavailable.
-              </span>
-            {/if}
-          </label>
-        {/if}
-        {#if !registrationSuccess}
-          <!-- Error when registration fails -->
-          <label for="registration" class="label mt-1">
-            <span class="label-text-alt text-error text-left">
-              There was an issue registering your account. Please try again.
-            </span>
-          </label>
-        {/if}
+      <div class="text-left mt-4">
+        <input
+          type="checkbox"
+          id="shared-computer"
+          class="peer checkbox checkbox-primary hover:border-base-100 rounded-lg border-2 border-base-100 transition-colors duration-250 ease-in-out inline-grid align-bottom checked:bg-base-100"
+        />
+        <!-- Warning when "This is a shared computer" is checked -->
+        <label
+          for="shared-computer"
+          class="cursor-pointer ml-1 text-sm grid-inline"
+        >
+          This is a public or shared computer
+        </label>
+        <label
+          for="registration"
+          class="label mt-4 !p-0 hidden peer-checked:block"
+        >
+          <span class="text-red-400 text-left">
+            In order to ensure the security of your private data, Webnative does
+            not recommend creating an account from a public or shared computer.
+          </span>
+        </label>
+      </div>
 
-        <div class="text-left mt-3">
-          <input
-            type="checkbox"
-            id="shared-computer"
-            class="peer checkbox checkbox-primary border-2 border-base-content hover:border-orange-300 transition-colors duration-250 ease-in-out inline-grid align-bottom"
-          />
-          <!-- Warning when "This is a shared computer" is checked -->
-          <label
-            for="shared-computer"
-            class="cursor-pointer ml-1 text-sm grid-inline"
-          >
-            This is a shared computer
-          </label>
-          <label
-            for="registration"
-            class="label mt-1 hidden peer-checked:block"
-          >
-            <span class="label-text-alt text-error text-left">
-              For security reasons, {appName} doesn't support shared computers at
-              this time.
-            </span>
-          </label>
-        </div>
+      <div class="flex items-center mt-4">
+        <a
+          class="!h-[52px] btn btn-outline !text-neutral-900 !border-neutral-900 !bg-neutral-50"
+          href="/connect"
```

Comment:
It looks like this should be updated to another route because `/connect` has been removed.

Reply:
fixed!